### PR TITLE
Pass through target attribute on continue link.

### PR DESCRIPTION
**Pull request by henrahmagix**
So links that should open in a new window can do.

@incuna/js Please merge, ta!

### CODE REVIEW

**Comment by henrahmagix** [comment]
Perhaps we could clone the original element so everything persists?

File: src/angular-external-link-interceptor.js [hunk 1]
```diff
@@ -94,6 +94,11 @@
                                 $scope.cancel = function () {
                                     $modalInstance.dismiss('cancel');
                                 };
+
+                                // Pass through target attribute, so links that
+                                // should open in a new window can.
+                                var element = angular.element(e.currentTarget);
+                                $scope.target = element.attr('target');
```
Comment (grahamgilchrist) [inline src/angular-external-link-interceptor.js]: An alternative would be to pass the `attributes` from the directive as parameters through `bindModal` and `externalModal`. But I think your solution here is neater :)

Comment (henrahmagix) [inline src/angular-external-link-interceptor.js]: The solution here was copied straight from some internal projects decorating the service, so it's not my solution but thankfully it's tested and production-worthy =)

I like your suggestion. In implementing this I started thinking about improving extensibility, but it started to get confusing – the controller needs access to the `event` parameter to `externalModal` – so I binned it for the moment.

Perhaps in cloning the source element and removing potentially offending attributes/values we can provide a better modal than one that only persists certain attributes. We could offer an attribute that takes an array of strings, each string being an attribute to remove when cloning the element, so the original anchor can have whatever attribute-directives it wants but also tell this service to not include them in the "Continue" button.

**Comment by henrahmagix** [comment]
Thanks!

**Comment by henrahmagix** [comment]
Released as a minor upgrade because it's new functionality. See https://github.com/incuna/angular-external-link-interceptor/releases/tag/1.2.0

**Pull request merged**